.github/workflows/module_integration.yml

```diff
@@ -85,112 +73,74 @@ jobs:
 
     - name: Run surfacebmi plus cfebmi
       run: ./cmake_build/ngen data/catchment_data.geojson "cat-27" data/nexus_data.geojson "nex-26" data/example_bmi_multi_realization_config.json
-      
-      - name: Build ISO C Fortran BMI
-        id: submod_build_2
-        uses: ./.github/actions/ngen-submod-build
-        with: 
-          mod-dir: "extern/iso_c_fortran_bmi/"
-      
-      - name: Build CFE
-        id: submod_build_3
-        uses: ./.github/actions/ngen-submod-build
-        with: 
-          mod-dir: "extern/cfe/"
-          targets: "cfebmi"
-
-      - name: Build Ngen
-        uses: ./.github/actions/ngen-build
-        with:
-          targets: "ngen"
-          build-cores: ${{ env.LINUX_NUM_PROC_CORES }}
-          #is this required for this test?
-          bmi_c: 'ON'
-          bmi_fortran: 'ON'
-        timeout-minutes: 15
-
-      - name: Run surfacebmi plus cfebmi
-        run: |
-          inputfile='data/example_bmi_multi_realization_config.json'
-          if [ ${{ runner.os }} == 'macOS' ] 
-          then
-              inputfile=data/example_bmi_multi_realization_config__macos.json
-          fi
-          ./cmake_build/ngen data/catchment_data.geojson "cat-27" data/nexus_data.geojson "nex-26" $inputfile
```

Comment:
Resolving the merge conflict this way is incorrect. Please review and understand https://github.com/NOAA-OWP/ngen/pull/407 . The above block and most of the other changes here effectively undoes the work in that PR. 

And please review the original state of @champham 's PR before you merged master... e.g. here: https://github.com/champham/ngen/compare/ded328...8d8ab23 to look at the actual changes that were intended. 

What should happen is that the tests that were added in the original PR (second link above) should be adjusted to use the refactored actions introduced in PR #407 (first link above), so that they can be merged in successfully.

Reply:
Above comments apply both to this file and `test_and_validate.yml`.